packages/@okta/vuepress-site/docs/guides/mfa/prerequisites/index.md

```diff
@@ -2,43 +2,18 @@
 title: Prerequisites
 ---
 
-Application developers and organizations are increasingly
-implementing multi-factor authentication to add an extra layer of
-security to their applications.
+It is quickly becoming the standard for application developers and organizations to implement multifactor authentication to add an extra layer of security to their applications. In this guide, we provide an example of how to use Okta's Multifactor Authentication (MFA) API to add MFA to your app. How you actually make the HTTPS calls depends on the programming language and web framework that your app uses.
```

Comment:
"to implement multifactor authentication (MFA) to" and then change the name of the API to the actual name, which is "Factors API".